janitor/functions/read_commandline.py

```diff
@@ -0,0 +1,77 @@
+import pandas_flavor as pf
+import pandas as pd
+from subprocess import Popen, PIPE
+import sys
+
+
+@pf.register_dataframe_method
+def read_commandline() -> pd.DataFrame:
```

Comment:
```suggestion
def read_commandline(cmd) -> pd.DataFrame:
```

The `cmd` should be the shell command that is desired.